### PR TITLE
Fix bad test detected by code cov tool

### PR DESCRIPTION
**Pull request by miiu96**
Fixed bad test detected by  code coverage tool

### CODE REVIEW

**Review by iulianpascalau — APPROVED** [review]

**Review by bogdan-rosianu — APPROVED** [review]

**Review by BeniaminDrasovean — APPROVED** [review]

**Review by iulianpascalau — CHANGES_REQUESTED** [review]
Will approve after the dependabot-fixes branch gets merged as that branch is protected

**Review by iulianpascalau — APPROVED** [review]

**Pull request merged**
